css/styles.css

```diff
@@ -0,0 +1,137 @@
+body {
+    margin: 0;
+}
+.topnav {
+    background-color:rgb(25, 50, 126);
+    overflow: hidden;
+    width: 100%;
+    height: 3em;
+}
+
+.topnav a {
+    float: left;
+    color: whitesmoke;
+    text-align: center;
+    padding-top: 1vh;
+    padding-right: 2vw;
+    text-decoration: none;
+    font-size: 1.2em;
+    text-shadow: 2px 2px black;
+}
+.topnav img {
+    float: left;
+    margin-top: 0.4vh;
+    margin-right: 0.5vw;
+    margin-left: 0.2vw;
+    width: 2em;
+    height: 4vh;
+}
+
+.topnav p {
+    float: left;
+    color: whitesmoke;
+    font-size: 2em;
+    margin-top: 0.4vh;
+}
+
+.topnav-right {
+    float: right; 
+}
+
+.twitter-filter-section p {
+    margin-top: 2.2vh;
+    margin-left: 8vw;
+    font-size: 1.2em;
+    float: left;
+    margin-right: 0.5vw;
+}
+
+.twitter-body .twitter-filter-section input {
+    margin-top: 2.2vh;
+}
+
+.twitter-dashboard-left {
+    width: 75%;
+    float: left;
+}
+
+.twitter-flex-container-dashboard {
+    display: flex;
+    align-content: flex-start;
+    width: 50%;
+    height: 100%;
+    flex-flow: row wrap;
+    position: relative;
+    left: -4.5vw;
+}
+
+.twitter-person {
+    width: 8.5vw;
+    height: 22vh;
+    text-align: center;
+    border: 0.2vw solid rgb(187, 184, 184) ;
+    border-radius: 3px;
+    margin-top: 1vh;
+    margin-right: 0.7vw;
+
+}
+.twitter-person input {
+    width: auto;
+    font-size: 0.9em;
+    margin-top: 0.5vh;
+    word-wrap: break-word;
+    min-width: 68.8px;
+}
+
+.twitter-person img {
+    width: 3vw;
+    height: 6vh;
+    padding-left: 20%;
```

Comment:
Q: what is the difference between margin and padding?

Reply:
Margin is the amount of space outside the border and padding is within the border

Reply:
correct